R/compute_weighted_intensity_levels.R

```diff
@@ -51,7 +51,7 @@
 #'
 #' # Add into a tibble with decreasing weight for older seasons
 #' data_input <- tibble::tibble(
-#'   observation = observations,
+#'   observed = observations,
```

Comment:
This is knitpicky, sorry.

In my opinion "observation" was more accurate as it is a noun rather than "observed" which is a verb.

Since we are talking about data, I think nouns are more appropriate while
I think verbs should be kept for functions.

Reply:
Keep or change as you like, I was just confused by the change.